components/typography/Paragraph.tsx

```diff
@@ -0,0 +1,31 @@
+import { ParagraphTypeStyle, ParagraphProps } from "@/types/typography/Paragraph";
+import { twMerge } from 'tailwind-merge';
+
+export default function Paragraph({
+  typeStyle = ParagraphTypeStyle.lg,
+  textColor = 'text-gray-700',
+  fontWeight,
+  className,
+  children,
+}: ParagraphProps) {
+  let classNames = '';
+
+  switch (typeStyle) {
+    case ParagraphTypeStyle.lg:
+      classNames = `text-lg ${fontWeight ?? ''} ${className || ''}`;
+      break;
+    case ParagraphTypeStyle.md:
+      classNames = `text-md ${fontWeight ?? ''} ${className || ''}`;
+      break;
+    case ParagraphTypeStyle.sm:
+      classNames = `text-sm ${fontWeight ?? ''} ${className || ''}`;
+      break;
+    default:
+      classNames = `text-lg font-regular ${className || ''}`;
+  }
+
+  return (
+    <p data-testid="Paragraph-test" className={twMerge(textColor, classNames)}>{children}</p>
```

Comment:
can you please explain this `datat-testId`

Reply:
I know it is of no use as of now and I removed it when I created the components. However, once the migration for all components is completed, we will add tests, so I left the data-testid as it

Reply:
If you are adding data-testid, it's good because later on it will be easier for us to integrate the tests related to it.

Reply:
I removed it 😅 let me add it again